serious/lib/site/views/rss.builder

```diff
@@ -51,12 +52,14 @@ xml.rss "xmlns:itunes" => "http://www.itunes.com/dtds/podcast-1.0.dtd", "xmlns:a
             selected_codec = @selected_audio_codec
           end
           url = article.audioformats[selected_codec]
-          type = "audio/x-#{selected_codec}"
```

Comment:
Seems like most of them are aliases. Did this actually cause issues?

Reply:
i have no idea @ltsavar suggested to change these because other people use things like `audio/mpeg` and stuff and i never seen this `x-` prefix

Reply:
i mostly see audio/mpeg but i there was no issue